dz15/src/app.js

```diff
@@ -0,0 +1,127 @@
+async function getInfoFromDummyjsonProductsCategories() {
+  const response = await fetch('https://dummyjson.com/products/categories')
+  const data = response.json()
+  return data
+}
+
+async function getCountFromDummyjsonProductsCategories() {
+  const counter = await getInfoFromDummyjsonProductsCategories()
+  return counter.length
+}
+
+async function findCategories(data, categories) {
+  const result = data.find((category) => category.name === categories)
+  if (result !== undefined) {
+    return result
+  }
+  return 0
+}
```

Comment:
```suggestion
function findCategoryByName(categories, categoryName) {
  return categories.find(({ name }) => name === categoryName)
}
```

- функция ищет категорию по названию, поэтому назвал findCategoryByName
- find вернёт или категорию, или undefined - нам подходит, 0 немного странновато возвращать)